src/test/java/teammates/test/cases/ui/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -54,7 +54,9 @@ public void testEditResponse() {
         assertEquals("Student 1 self feedback.",fr.getResponseDetails().getAnswerString());
      
         submitPage = loginToInstructorEditStudentFeedbackPage("IESFPTCourseinstr", "student1InIESFPTCourse@gmail.tmt", "session1InIESFPTCourse");
-        submitPage.verifyHtml("/InstructorEditStudentFeedbackPageOpen.html");
+
+        // Full HTML verification already done in InstructorFeedbackSubmitPageUiTest
+        submitPage.verifyHtmlMainContent("/InstructorEditStudentFeedbackPageOpen.html");
```

Comment:
This is changed, but the HTML file is untouched. God mode doesn't touch the file if the parts to be compared is the same (even if the HTML file is full file, if we pass to `verifyHtmlMainContent`, God mode will only compare the `mainContent` part).
If you want God mode to regenerate it, mess up with the `mainContent` part (can be as simple as adding random characters) then regenerate.